### PR TITLE
fix(route): 78动漫

### PR DESCRIPTION
**Pull request by nczitzk**
## Involved Issue / 该 PR 相关 Issue

Close #

## Example for the Proposed Route(s) / 路由地址示例

```routes
/78dm/news
```

## New RSS Route Checklist / 新 RSS 路由检查表
  
- [ ] New Route / 新的路由
  - [ ] Follows [Script Standard](https://docs.rsshub.app/joinus/advanced/script-standard) / 跟随 [路由规范](https://docs.rsshub.app/zh/joinus/advanced/script-standard)
- [ ] Anti-bot or rate limit / 反爬/频率限制
  - [ ] If yes, do your code reflect this sign? / 如果有, 是否有对应的措施?
- [ ] [Date and time](https://docs.rsshub.app/joinus/advanced/pub-date) / [日期和时间](https://docs.rsshub.app/zh/joinus/advanced/pub-date)
  - [ ] Parsed / 可以解析
  - [ ] Correct time zone / 时区正确
- [ ] New package added / 添加了新的包
- [ ] `Puppeteer`

### CODE REVIEW

File: lib/routes/78dm/namespace.ts [hunk 1]
```diff
@@ -3,5 +3,7 @@ import type { Namespace } from '@/types';
 export const namespace: Namespace = {
     name: '78 动漫',
     url: '78dm.net',
+    categories: ['new-media'],
```
Comment (TonyRL) [inline lib/routes/78dm/namespace.ts:6]:
```suggestion
    categories: ['anime'],
```

File: lib/routes/78dm/index.ts [hunk 2]
```diff
@@ -11,77 +10,464 @@ import { parseDate } from '@/utils/parse-date';
...
+            target: '/ht_list/133/0/0/1',
+        },
+        {
+            title: '好贴推荐 - 民用模型',
+            source: ['78dm.net/ht_list/134/0/0/1.html'],
+            target: '/ht_list/134/0/0/1',
+        },
+        {
+            title: '好贴推荐 - 兵人模型',
+            source: ['78dm.net/ht_list/135/0/0/1.html'],
```
Comment (TonyRL) [inline lib/routes/78dm/index.ts:469]: All sources should start with `www.78dm.net`

File: lib/routes/78dm/index.ts [hunk 3]
```diff
@@ -11,77 +10,464 @@ import { parseDate } from '@/utils/parse-date';
...
+        image,
+        author: $('meta[property="og:site_name"]').prop('content'),
+        language,
     };
-}
+};
+
+export const route: Route = {
+    path: '/:category{.+}?',
+    name: '78 动漫',
```
Comment (TonyRL) [inline lib/routes/78dm/index.ts:139]: Route name should not be the same as namespace name.

File: lib/routes/78dm/namespace.ts (hunk 1 above)
Comment (nczitzk) [reply lib/routes/78dm/namespace.ts:6]: 👌

**Pull request merged**